web/src/ShareSnapshotModal.tsx

```diff
@@ -12,62 +12,108 @@ type props = {
   isOpen: boolean
 }
 
-// TODO(dmiller): in the future this should also show if you are logged in and who you are
 export default class ShareSnapshotModal extends PureComponent<props> {
   render() {
+    let tcUser = this.props.tiltCloudUsername
+    let user = tcUser ? tcUser : "[anonymous user]"
     let link = this.renderShareLink()
     return (
       <Modal
         onRequestClose={this.props.handleClose}
         isOpen={this.props.isOpen}
         className="ShareSnapshotModal"
       >
-        <div className="ShareSnapshotModal-pane">{link}</div>
+        <h2 className="ShareSnapshotModal-title">Share a Shapshot</h2>
+        <div className="ShareSnapshotModal-pane">
+          <p>
+            Let anyone explore your current Tilt session with an interactive
+            snapshot.
```

Comment:
i don't feel strongly but maybe this gets more clearly at the fact that the state, while interactive, is static:
"Let anyone explore your dev environment with an interactive snapshot of your current Tilt state."

Reply:
I was playing with "session" vs "state" and you just nudged me into thinking the latter may be better.